src/rendering/renderers/CrosshairRenderer.ts

```diff
@@ -44,32 +61,41 @@ export class CrosshairRenderer extends BaseRenderer {
         return CrosshairRenderer.CROSSHAIR_ID
     }
 
+    public render(context: RenderingContext, done: DoneFunction): void {
+        this.lastContext = context
+
+        super.render(context, done)
+    }
+
+    protected handleMouseEvent(event: MousemoveEvent | MouseleaveEvent): void {
+        this.lastEvent = event
+
+        if (!this.lastContext) return
+        this.render(this.lastContext, () => { })
+    }
+
     protected update(
         context: RenderingContext,
         container: Container,
     ): Container {
```

Comment:
I think idea should be next
you create a single cross graphic on update and subscribe to `context.application.resizeTo` 

```
    protected update(
        context: RenderingContext,
        container: Container,
    ): Container {
      this.lastContext = context
    
      if (!context.application.resizeTo) {
        this.clear()
        return container
      }
      
      if (this.eventTarget !== context.application.resizeTo) {
            this.clear()
            this.eventTarget.removeEventListener('pointermove', this.updatePointer)
            this.eventTarget = context.application.resizeTo
            this.eventTarget.addEventListener('pointermove', this.updatePointer)
      }
      
      // create pointer should create 
      const [pointer, pointerState] = this.get('pointer', () => this.createPointer())
      if (pointerState.new) container.add(pointer)
      
      return container
    }
```